src/row.js

```diff
@@ -62,6 +62,22 @@ export class Row {
     this.enemyFaction = row[47];
   }
 
+  getComposition = brackets => {
+    // We could have added all 5 classes and then filter out falsy values, but it would not be resilient to "ghost player" bugs
+    let arr = [this.enemyPlayerClass1, this.enemyPlayerClass2];
+    if (brackets.includes('5s'))
```

Comment:
always put braces arround if (for readability) even if js supports no braces for one line statements